test/ClangImporter/anonymous.swift

```diff
@@ -0,0 +1,3 @@
+// RUN: %target-swift-frontend -Xcc -fms-extensions -import-objc-header %S/Inputs/anonymous.h -typecheck -verify %s
```

Comment:
File naming nit: Because there are also C-standard anonymous declarations and we're not testing those here, I'd recommend giving the test (and header) a name reflecting that it's a Microsoft extension.

I notice that C-standard anonymous structs are tested by `testAnonStructs()` in test/ClangImporter/ctypes_parse.swift. Maybe ctypes_parse_msvc.swift would be a good name?

Reply:
Sure, Im fine with that.